README.md

```diff
@@ -19,7 +19,7 @@ Create new error instances with a code and additional properties.
 
 ## Installation
 
```

Comment:
Can you keep the space so that it says on a new line?

Reply:
Sorry, my editor keeps removing white space from the end of lines.  I've changed it to be a `console` code block.